code/__DEFINES/flags.dm

```diff
@@ -24,6 +24,7 @@
 #define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
 
 #define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define SEMIOPENCONTAINER	8192

```

Comment:
It's about logic and understanding, making the message clear. You're referring to glass containers here, they are not open, they are transparent, and that is the reason you can see their reagents.